kolibri/plugins/style_guide/assets/src/views/content/components/simple-modals.vue

```diff
@@ -0,0 +1,302 @@
+        Modals should be keyboard accessible for users who use assistive
+        technology.
+      </p>
+      <p>
+        Tabbing should limit the user to the options within a modal only and not
+        extend to the UI beneath it.
+      </p>
+
+      <h3>URL behavior</h3>
+      <p>
+        Simple modals should not have a URL (i.e. if user refreshes page, or uses
+        "back" in browser, it returns them to the page below the modal).
+      </p>
+
+      <h3>Dismissing modals</h3>
+      <p>
+        Modals can be dismissed in 2 ways: through a "cancel/close" button within
+        the modal or pressing ESC.
+      </p>
+      <p>
+        Modals are never dismissible by clicking the backdrop.
+      </p>
+
+      <h3>Error states</h3>
+      <p>
+        See modal slots below.
+      </p>
+
+      <h3>Internationalization</h3>
+      <p>
+        If a button group happens to exceed the width of a modal, the buttons
+        should wrap, with the secondary action appearing below the primary action.
+        Buttons should still right-align.
+      </p>
+
+      <h3>Layout</h3>
+      <p>
+        Modals have 3 sections that are always present: title, content, and
+        actions.
+      </p>
+      <p>Optional:</p>
+      <ul>
+        <li>Text prop below title</li>
+        <li>Secondary button</li>
+      </ul>
+
+      <h4>Title</h4>
+      <p>
+        Keep this short, concise, and easy for the user to understand. Should not
+        be structured as a sentence or question.
+      </p>
+
+      <h4>Content</h4>
+      <p>
+        Along with text, the following UI elements are allowed in the content
+        section of a modal:
+      </p>
+      <ul>
+        <li>Text field</li>
+        <li>Checkboxes</li>
+        <li>Radio</li>
+        <li>Select dropdown</li>
+        <li>List with vertical scrollbar</li>
+      </ul>
+
+      <h4>Actions</h4>
+      <p>
+        There is always a raised action button present at the bottom right. Should
+        always be affirmative (i.e. will save data or continue user to the next
+        step).
+      </p>
+      <p>
+        If there is a need for a secondary action, use a flat secondary button
+        that is placed to the left of the action button. Secondary actions should
+        always be dismissive (i.e. does not save any altered data, or cancels a
+        next step).
+      </p>
+      <p>
+        The button group should be right-aligned.
+      </p>
+
+      <h4>Do not</h4>
+      <ul>
+        <li>
+          Use buttons or other actions in the content section. If it's needed, the
+          UX may need to be changed.
+        </li>
+        <li>
+          Present unclear or ambiguous actions. Actions should be directly related
+          to the title and content in the modal.
+        </li>
+        <li>
+          Use an 'X' button. Avoid creating redundant actions.
+        </li>
+        <li>
+          Use a back button, or otherwise create a multi-step process.
```

Comment:
Can remove "or otherwise create a multi-step process" and just have "Use a back button"